css-positioning/fixed.css

```diff
@@ -0,0 +1,11 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  top: 0;
+  right: 700px;
```

Comment:
You can also set both `right` to 0 and `left` to 0. It will center it on the page for you